_posts/2018-09-29-analyzing-the-behaviour-of-python-function-splice.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: post
+title: "Analyzing the behaviour of Python function slice"
+date: "2018-09-29 10:17:46 +0530"
+tag:
+  - python
+  - slice
+  - python trick questions
+---
+
+
+![Title Image](/assets/images/python_slice_function/title_image.jpg)
+
+Last Friday, I was sitting at one of the good coffee shop of Bangalore with my
+friend. Coffee and discussion is the best combination to release stress. It was
+looking like a perfect Friday eve, until my friend stroke with an idea of asking
+me a question. Suddenly He said, "What if I ask you one quiz question based on
+Python?". "Aha! What is the level of difficulty?" I confusingly requestioned.
+Here he played the trick. He said, "It is a bit difficult, but not that much
```

Comment:
The "much" there I don't think makes sense. Maybe "not that difficult" or "not very difficult" or "not too difficult"?